Tasks/DotNetCoreCLIV2/dotnetcore.ts

```diff
@@ -455,6 +456,25 @@ export class dotNetExe {
             tl.debug(`Unable to log NuGet task init telemetry. Err:( ${err} )`);
         }
     }
+
+    private setUpConnectedServiceEnvironmentVariables() {
+        var connectedService = tl.getInput('ConnectedServiceName');
+        if(connectedService) {
+            var authScheme: string = tl.getEndpointAuthorizationScheme(connectedService, false);
+            if (authScheme && authScheme.toLowerCase() == "workloadidentityfederation") {
+                process.env.AZURESUBSCRIPTION_SERVICE_CONNECTION_ID = connectedService;
+                process.env.AZURESUBSCRIPTION_CLIENT_ID = tl.getEndpointAuthorizationParameter(connectedService, "serviceprincipalid", false);
+                process.env.AZURESUBSCRIPTION_TENANT_ID = tl.getEndpointAuthorizationParameter(connectedService, "tenantid", false);
+                tl.debug('Environment variables AZURESUBSCRIPTION_SERVICE_CONNECTION_ID,AZURESUBSCRIPTION_CLIENT_ID and AZURESUBSCRIPTION_TENANT_ID are set');
+            }
+            else {
+                tl.debug('Connected service is not of type Workload Identity Federation');
```

Comment:
should be this error/or warning for better visibility from customer perspective?

Reply:
I have updated the message to warning.